docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -55,4 +54,4 @@ ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tag
 * Install Nagios
 * Print URL and credentials for Jenkins and Nagios
 
-NOTE: All tasks are idempotent. You can run them multiple times and they will produce the same results.
\ No newline at end of file
+//NOTE: All tasks are idempotent. You can run them multiple times and they will produce the same results.
```

Comment:
same?

Reply:
yes, for now. noted in https://issues.jboss.org/browse/RHMAP-18949

Reply:
resolved